website/tutorial.md

```diff
@@ -32,13 +32,14 @@ variables the site needs.
 Open <walkthrough-editor-open-file filePath="././ui/src/environments/environment.prod.ts">
 environment.prod.ts</walkthrough-editor-open-file>
 
-Fill in the GTM Web Container ID and change locale settings accordingly.
+Fill in the GTM Web Container ID and change local settings accordingly.
```

Comment:
This should be "locale", [see here](https://developer.mozilla.org/en-US/docs/Glossary/Localea).